Makefile

```diff
@@ -0,0 +1,22 @@
+SHELL := /usr/bin/env bash
+
+DOCKERFILE = ./Dockerfile
+
+# Include shared Makefiles
+include project.mk
+include standard.mk
+
+default: generate-template docker-build
+
+CONTAINER_ENGINE?=docker
+
+.PHONY: docker-build
+docker-build: build
+
+.PHONY: generate-template
+generate-template:
+	if [ "${IN_CONTAINER}" == "true" ]; then \
+		$(CONTAINER_ENGINE) run --rm -v `pwd -P`:`pwd -P` python:2.7.15 /bin/sh -c "cd `pwd`; pip install oyaml; `pwd`/${GEN_TEMPLATE}"; \
```

Comment:
We _really_ shouldn't be using py2 anywhere anymore.

Reply:
Hey Eric,

I'm checking with James to see if using python3 for the `generate_template.py` script will break anything with Jenkins. I'm going to merge this commit for now, and I'll come back and switch this over to python3 once I hear back from him.